src/libp2p/Libp2p.Protocols.Noise/NoiseProtocol.cs

```diff
@@ -29,8 +29,8 @@ public class NoiseProtocol(MultiplexerSettings? multiplexerSettings = null, ILog
     {
         StreamMuxers =
         {
-            multiplexerSettings is null || multiplexerSettings.Multiplexers.Any() ? ["na"] : [.. multiplexerSettings.Multiplexers.Select(proto => proto.Id)]
-        }
+           multiplexerSettings is null ? ["na"] : multiplexerSettings.Multiplexers.Any() ? [.. multiplexerSettings.Multiplexers.Select(proto => proto.Id)] : []
```

Comment:
Old code has `multiplexerSettings.Multiplexers.Any()`, which is not correct looks like
Seems like it should be `!multiplexerSettings.Multiplexers.Any()`

na means no protocol available and it is better to return it instead of empty array in any way